test/test_stochastic.py

```diff
@@ -90,19 +90,19 @@ def test_stochastic_asgd():
 
 def test_sag_asgd_sinkhorn():
     # test all algorithms
-    n = 15
+    n_samples = 15
     reg = 1
     nb_iter = 100000
```

Comment:
why not rename this to num_iter_max as well? (or maxiter, depending on what we choose)